controller/relay-image.go

```diff
@@ -24,26 +26,27 @@ func relayImageHelper(c *gin.Context, relayMode int) *OpenAIErrorWithStatusCode
 	group := c.GetString("group")
 
 	var imageRequest ImageRequest
+
 	if consumeQuota {
 		err := common.UnmarshalBodyReusable(c, &imageRequest)
 		if err != nil {
 			return errorWrapper(err, "bind_request_body_failed", http.StatusBadRequest)
 		}
 	}
 
-	// Prompt validation
-	if imageRequest.Prompt == "" {
-		return errorWrapper(errors.New("prompt is required"), "required_field_missing", http.StatusBadRequest)
+	// Model validation
+	if imageRequest.Model != "" && imageRequest.Model != "dall-e-3" {
+		imageModel = "dall-e-2"
 	}
 
-	// Not "256x256", "512x512", or "1024x1024"
-	if imageRequest.Size != "" && imageRequest.Size != "256x256" && imageRequest.Size != "512x512" && imageRequest.Size != "1024x1024" {
-		return errorWrapper(errors.New("size must be one of 256x256, 512x512, or 1024x1024"), "invalid_field_value", http.StatusBadRequest)
+	// Size validation
+	if imageRequest.Size != "" {
+		requestSize = imageRequest.Size
 	}
 
-	// N should between 1 and 10
```

Comment:
missed N validation here. N should between 1 and 10 in `dall-e-2`, and only `n=1` is supported in `dall-e-3`